src/main/cpp/SwerveModule.cpp

```diff
@@ -81,37 +74,6 @@ bool SwerveModule::Reset() {
     this->steer_pid_->Reset();
 
     //TODO reset sensor readings
-    //TODO reset WPILib kinematics stuff
-    return true;
-}
-
-bool SwerveModule::GetLocationOnRobot(frc::Translation2d *trans) {
-    OKC_CHECK(trans != nullptr);
-
-    *trans = this->trans_;
-
-    return true;
-}
-
-bool SwerveModule::GetSwerveModulePosition(frc::SwerveModulePosition *pos) {
-    OKC_CHECK(pos != nullptr);
-
-    *pos = this->pos_;
-
-    return true;
-}
-
-bool SwerveModule::GetSwerveModuleState(frc::SwerveModuleState *state) {
-    OKC_CHECK(state != nullptr);
-
-    *state = this->state_;
-    
-    return true;
-}
```

Comment:
do we not need this deleted code?

Reply:
That was for the WPILib swerve support, which I gave up on using, because a) it needed the units library which made trying to do anything confusing and overly long winded, b) it was bad, and c) it works without it. I probably should've opened up a separate PR for feat/swerve-tuning or whatever. . .